### PR TITLE
Add better solution for cross-platform font renderings

### PR DESCRIPTION
**Pull request by jiraiyame**
this feature will support in next generation css, ref: https://drafts.csswg.org/css-fonts-4/#system-ui-def

### CODE REVIEW

**Comment by Hypnosphi** [comment]
In fact, it can be just `font-family: system-ui`, because we have cssNext in the project: http://cssnext.io/features/#system-ui-font-family

**Comment by jiraiyame** [comment]
OK, i am missing that. One more thing, consider whether it is necessary to keep the font like `sans-serif` as a general fallback mechanism?

**Comment by Hypnosphi** [comment]
I think it won't hurt to keep it

**Comment by jiraiyame** [comment]
@Hypnosphi Changed done!

File: components/global/global.css [hunk 1]
```diff
@@ -33,7 +33,7 @@
 .font {
   color: text-color;
 
-  font-family: -apple-system, BlinkMacSystemFont, "Helvetica Neue", Arial, sans-serif;
+  font-family: -apple-system, Arial, sans-serif;
```
Comment (Hypnosphi) [inline components/global/global.css]: I guess `-apple-system` is a typo =)

Comment (jiraiyame) [inline components/global/global.css]: wait a moment, i also find this mistake😅

**Comment by jiraiyame** [comment]
@Hypnosphi this is the final change i guess 😆

**Comment by Hypnosphi** [comment]
Thank you, looks great (in any sense)!

**Comment by Leonya** [comment]
@jiraiyame unfortunately, we are going to revert this PR – the list of fonts we had before was a carefully considered one and we don't want any unexpected additions. For example, we have tested Segoe UI about a year ago and decided it's not a good fit for our UIs.

**Pull request merged**